src/framework/core/graphicalapplication.h

```diff
@@ -115,6 +115,12 @@ class GraphicalApplication : public Application
     float getStaticTextScale() const { return m_staticTextScale; }
     void setStaticTextScale(float v) { m_staticTextScale = v; }
 
+    float getEffectAlpha() const { return m_effectAlpha; }
+    void setEffectAlpha(float v) { m_effectAlpha = v; }
+
+    float getMissileAlpha() const { return m_missileAlpha; }
+    void setMissileAlpha(float v) { m_missileAlpha = v; }
+
```

Comment:
This can be moved to client.h, because it is more a client feature other than part of graphical application

We want to be
g_client.getEffectAlpha
g_client.setEffectAlpha

etc